contributed_definitions/NXdispersive_material.nxdl.xml

```diff
@@ -0,0 +1,170 @@
+<?xml version="1.0" ?>
+<?xml-stylesheet type="text/xsl" href="nxdlformat.xsl"?>
+<definition xmlns="http://definition.nexusformat.org/nxdl/3.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://definition.nexusformat.org/nxdl/3.1 ../nxdl.xsd" type="group" category="application" name="NXdispersive_material" extends="NXobject">
+    <doc>
+         NXdispersion
+    </doc>
+    <group type="NXentry">
+        <field name="definition" type="NX_CHAR">
+            <doc>
+                 An application definition for a dispersive material.
+            </doc>
+            <attribute name="version" type="NX_CHAR">
+                <doc>
+                     Version number to identify which definition of this application definition was
+                     used for this entry/data.
+                </doc>
+            </attribute>
+            <attribute name="url" type="NX_CHAR">
+                <doc>
+                     URL where to find further material (documentation, examples) relevant to the
+                     application definition
+                </doc>
+            </attribute>
+            <enumeration>
+                <item value="NXdispersive_material"/>
+            </enumeration>
+        </field>
+        <field name="chemical_formula" type="NX_CHAR"/>
```

Comment:
also documentation is missing

Reply:
There actually is no NXsample in this definition as this does not really refer to a real sample. It just is a optical model for a material. This is why I decided to add the `chemical_formula` directly here. Eventually, this should also be usable in a process within an NXellipsometry measurement, where this will be reflected by NXsample within the NXellipsometry definition.

Reply:
Ok I figured that there is no reason to not create NXsample here and just fill it with sparse info. However, I still am not sure if I should add the phase information as I do or if I rather should use the field `state` in NXsample.